README.md

```diff
@@ -35,10 +35,28 @@ Installing and building:
 Running:
 
     $ prom2json http://my-prometheus-client.example.org:8080/metrics
+    $ prom2json < /tmp/metrics.txt
+    $ prom2json /tmp/metrics.txt
     
 Running with TLS client authentication:
 
     $ prom2json --cert=/path/to/certificate --key=/path/to/key http://my-prometheus-client.example.org:8080/metrics
+    
+Running without TLS validation:
```

Comment:
Could you add a big and fat warning? Something like:

Running without TLS validation (**insecure!**):